UC-Berkeley-Extension/global
# Token test

## Asana Ticket(s)
- See ticket [Token Test](https://app.asana.com/0/0/1198516086429365/f).

## Description

* Adds the Allowed Filters and Token Filter module
* Adds the token filter to the Plain Text and Restricted HTML text format.
* Sets a test article field to only allow the Plain Text filter.
* Let's you demo the feature.

## Testing instructions
1. Checkout this branch
1. Run `composer install`
2. Build the site
    * From inside the VM, run `/var/www/html/scripts/setup.sh` or `refresh.sh`
1. Create an Article
1. For the `Test Field` set the value to `Hi there!`
1. For the `Another Test Field` use the token browser to set the field value to `Test Field` or paste in `[node:field_test_field]`
1. Save the node
1. Observe that the `Another Test Field` output matches that of `Test Field`
1. Edit the node. 
1. Add some other content to the field, such as `Test [node:field_test_field] test`
1. Save the node
1. Observe that the `Another Test Field` output matches that of `Test [Test Field content] test`


## Relevant Screenshots/gifs:
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.

## Remaining tasks

- [ ] Delete these test fields
- [ ] Configure the filter as needed
- [ ] Configure permissions for the filter.
